keps/sig-scheduling/4664-guarantee-pdb-when-preemption-happens/kep.yaml

```diff
@@ -1,7 +1,8 @@
 title: Guarantee PodDisruptionBudget When Preemption Happens
-kep-number: 3280
+kep-number: 4664
```

Comment:
This is an unusual change to make. Let's add explanatory comments to the YAML.

Reply:
Agree.